Releases.md

```diff
@@ -4,11 +4,16 @@ See [DirectML version history on MSDN](https://docs.microsoft.com/windows/win32/
 
 | Version                | Feature level                                                                                                                      | First available in (Windows 10)                          | Redistributable                                                                           |
 | ---------------------- | ---------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- | ----------------------------------------------------------------------------------------- |
-| [1.4.1](#directml-141) | [DML_FEATURE_LEVEL_3_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_3_0) | Not yet released                                         | [Microsoft.AI.DirectML.1.4.0](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.4.1) |
+| [1.4.2](#directml-142) | [DML_FEATURE_LEVEL_3_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_3_0) | Not yet released                                         | [Microsoft.AI.DirectML.1.4.2](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.4.2) |
+| [1.4.1](#directml-141) | [DML_FEATURE_LEVEL_3_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_3_0) | Not yet released                                         | [Microsoft.AI.DirectML.1.4.1](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.4.1) |
```

Comment:
1.4.0 and 1.4.1 will never release to Windows at this point, so perhaps change to "redistributable only" for these builds?

Reply:
Sure. I like "redistributable only".

Reply:
Oh, hmm. Actually Iron is _1.4.0_. It's really 1.4.1 and 1.4.2 which will never be released, because they contain changes (the metacommand fixes and the redist-specific PIX changes) that don't meet the bar for the OS.